source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -56,6 +56,7 @@
         }
         this.save = function () {
             var updateCalificators = this.calificatorsManagerModel.toJs();
+            var updateEvaluators = this.calificatorsManagerModel.toEvaluators();
```

Comment:
@andresmoschini  I believe that 'updateEvaluatorsNames' it is also more representative, because the toJS() method return evaluators, but this method updates the evaluators after being modified